magefile.go

```diff
@@ -1333,7 +1333,7 @@ func collectPackageDependencies(platforms []string, packageVersion string, packa
 			}
 		}
 	} else {
-		archivePath = movePackagesToArchive(dropPath, platforms, packageVersion, nil)
+		archivePath = movePackagesToArchive(dropPath, platforms, packageVersion, dependencies)
```

Comment:
I see 👀

Reply:
Yep :sweat_smile: 
Bad refactoring of `movePackagesToArchive` passed `nil ` instead of the actual list of dependencies